packages/xo-web/src/xo-app/settings/remotes/index.js

```diff
@@ -300,6 +318,19 @@ export default decorate([
         remote,
       }),
     },
+    computed: {
+      remoteWithInfo: (_, { remotes, remotesInfo }) => {
+        if (remotesInfo) {
+          for (const remoteType in remotes) {
+            remotes[remoteType].map(remote => {
```

Comment:
Never, ever, mutate a value in the state or props.

A computed must be a new value, computed from values in state and/or props.